doc/nrf/doc_build.rst

```diff
@@ -16,20 +16,20 @@ Before you can build the documentation, install the |NCS| as described in :ref:`
 Make sure that you have installed the required :ref:`Python dependencies <additional_deps>`.
 
 See :ref:`zephyr:documentation-processors` in the Zephyr developer guide for information about installing the required tools to build the documentation and their supported versions.
-In addition to these tools, you must install `mscgen`_ and make sure the ``mscgen`` executable is in your ``PATH``.
+In addition to these tools, you must install `mscgen`_ and make sure the :file:`mscgen` executable is in your ``PATH``.
 
 .. note::
-   On Windows, the Sphinx executable ``sphinx-build.exe`` is placed in the ``Scripts`` folder of your Python installation path.
-   Dependending on how you have installed Python, you might need to add this folder to your ``PATH`` environment variable.
+   On Windows, the Sphinx executable :file:`sphinx-build.exe` is placed in the :file:`Scripts` folder of your Python installation path.
```

Comment:
Again, sphinx-build.exe refers to the command/executable, not really the file (even though here it's okay-ish, since it's also the file name).

Reply:
I'll revert to code.